ts/db.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Matt Tracy (matt.r.tracy@gmail.com)
+
+package ts
+
+import (
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/proto"
+)
+
+// DB provides Cockroach's Time Series API.
+type DB struct {
+	kv *client.KV
+}
+
+// NewDB creates a new DB instance.
+func NewDB(kv *client.KV) *DB {
+	return &DB{
+		kv: kv,
+	}
+}
+
+// storeData attempts to store the supplied time series data on the server.
```

Comment:
Phrase comments more assertively Matt! The word "attempts" here seems an unnecessary qualification considering the function returns an error.